flexget/plugins/output/plugin_decompress.py

```diff
@@ -227,12 +220,12 @@ def handle_entry(self, entry, config):
 
                     log.verbose('Extracted: %s' % path )
                     success = True
-                except (IOError, os.error) as e:
+                except (IOError, os.error) as error:
                     error_message = 'OS error while creating file: %s (%s)' % \
-                                    (destination, e)
-                except (zipfile.BadZipfile, rarfile.Error) as e:
+                                    (destination, error)
+                except (zipfile.BadZipfile, rarfile.Error) as error:
```

Comment:
Can this even happen? Aren't they already caught by the try blocks in the `open_archive` function?